src/components/components/HeroSection/HeroSection.scss

```diff
@@ -0,0 +1,101 @@
+.HeroSection {
+	padding-top: 5%;
+	max-width: 1260px;
+	width: 80%;
+	max-height: 673px;
+	height: 80vh;
+	margin: 0 auto;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: 3%;
+	font-family: var(--inter);
+
+	&__info {
+		padding: 30px 0;
+	}
+
+	&__info,
+	&__image {
+		flex-basis: 50%;
+		max-width: 50%;
+	}
+
+	&__title {
+		margin: 0;
+		font-size: calc(32px + (56 - 32) * ((100vw - 768px) / (1280 - 768)));
+		line-height: 133%;
+		font-weight: 700;
+		color: var(--color-bunting);
+
+		&--circled {
+			padding: 0 10px;
+			border: 3px solid var(--color-blue-ribbon);
+			border-radius: 30px;
+		}
+	}
+
+	&__text {
+		font-size: calc(16px + 2 * (100vw / 1280));
+		color: #7d7cac;
+		line-height: 150%;
+	}
+
+	&__button {
+		text-decoration: none;
+		display: inline-block;
+		padding: 10px 20px;
+		color: var(--color-white);
+		font-size: 14px;
+		line-height: 16px;
+		background-color: var(--color-blue-ribbon);
+		border: 2px solid var(--color-blue-ribbon);
+		border-radius: 8px;
+	}
+}
+
+.HeroSection__image {
+	& img {
+		max-width: 100%;
+	}
+}
+
+@media only screen and (min-width: 1280px) {
+	.HeroSection__title {
+		font-size: 56px;
+	}
+}
+
+@media (max-width: 425px) {
+	.HeroSection {
+		flex-direction: column;
+		width: 100%;
+		margin-top: 25px;
+
+		&__info {
+			margin: 25px 25px 0 25px;
+			padding: 0;
+		}
+
+		&__info,
+		&__image {
+			flex-basis: 50%;
```

Comment:
don't need the line 82 I guess since its in line 20

Reply:
true, thanks!